packages/closer/pages/affiliate/index.tsx

```
@@ -0,0 +1,252 @@
+  useEffect(() => {
+    const loadData = async () => {
+      if (!totalEarningsFilter) return;
+      try {
+        // TODO: figure out where is the data coming from
+        // const result = await Promise.all([
+        // ]);
+        // setTokensSales(result[0].results);
+      } catch (err) {
+        console.error("Error loading data:", err);
+      }
+    };
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)